docs/faq.rst

```diff
@@ -0,0 +1,225 @@
+*******************************************************************************
```

Comment:
This topic is missing the metadata. 
Must be added.

Reply:
Added metadata